lib/git_reflow/commands/review.rb

```diff
@@ -6,15 +6,38 @@
   c.flag [:t, :title], default_value: 'last commit message'
   c.flag [:m, :message], default_value: 'title'
   c.action do |global_options,options,args|
-    review_options = {
-      'base' => args[0],
-      'title' => global_options[:title],
-      'body' => global_options[:message]
-    }
 
-    review_options['title'] ||=  GitReflow.get_first_commit_message
-    review_options['body']  ||=  review_options['title']
+    pull_request_msg_file = '.git/GIT_REFLOW_PR_MSG'
+    editor = true
 
-    GitReflow.review review_options
+    if global_options[:title] || global_options[:message]
+      review_options = {
+        'base' => args[0],
+        'title' => (global_options[:title]   || GitReflow.current_branch),
+        'body' =>  global_options[:message]
+      }
+      editor = false
```

Comment:
it still seems strange to have this here... why can't we just change line 22 into an `else`?